docs/decisions/002-otel-tracing.md

```
@@ -0,0 +1,14 @@
+# ADR 002: OpenTelemetry para traces/métricas
+
+## Contexto
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)